edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/MetricsScraper.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics
     using System.Text.RegularExpressions;
     using System.Threading;
     using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Util;
```

Comment:
>using Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Util; [](start = 4, length = 58)

still needed?

Reply:
yes, for PrometheusMetricsParser on line 48